src/com/pentalog/pentastagiu/homework2/Shop/Basket.java

```diff
@@ -0,0 +1,38 @@
+package com.pentalog.pentastagiu.homework2.Shop;
+
+public class Basket {
+    private Customer customer;
+    private double totalPrice;
+
+    public Basket(Customer customer) {
+        this.customer = customer;
```

Comment:
ar trebui facuta aici si o validare pt customer, sa nu fie null; fara validare, cineva poate crea un basket asa: new Basket(null) iar cand ar invoca getTotalPrice, la linie 14 va fi aruncat un NullPointerException
pt validare de referinta nenula, poti folosi 
`Objects.requireNonNull(customer, "A customer should be provided when creating a basket");`